content/en/docs/started/workstation/minikf-gcp.md

```diff
@@ -2,16 +2,6 @@
 title = "Deploy using MiniKF on GCP"
 description = "Using Google Cloud Marketplace to deploy MiniKF (mini Kubeflow) on Google Cloud Platform (GCP)"
```

Comment:
```suggestion
title = "Deploy with MiniKF on Google Cloud"
description = "Using Google Cloud Marketplace to deploy MiniKF (mini Kubeflow) on Google Cloud"
```

- It was "deploy using MiniKF..." (title) vs "using ... Marketplace to deploy... " (description), so I'm suggesting a tweak here. LMKWYT 🙂
- Also, GCP has rebranded, I think